easybuild/easyconfigs/f/FLTK/FLTK-1.3.5-foss-2019a.eb

```diff
@@ -0,0 +1,41 @@
+#
+# author: Dina Mahmoud Ibrahim ( Cairo University )
+#
+easyblock = 'ConfigureMake'
+
+name = 'FLTK'
+version = '1.3.5'
+
+homepage = 'https://www.fltk.org'
+description = """FLTK is a cross-platform C++ GUI toolkit for UNIX/Linux (X11), Microsoft Windows,
+ and MacOS X. FLTK provides modern GUI functionality without the bloat and supports 3D graphics via OpenGL
+ and its built-in GLUT emulation."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
@crubb Why not use the existing `FLTK-1.3.5-GCC-8.2.0-2.31.1.eb`?

Reply:
Didn't see it, sorry. Fixed!